### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by yveskerwyn**

### CODE REVIEW

File: docs/interacting/commands/disk.md [hunk 1]
```diff
@@ -44,15 +44,17 @@ Arguments:
...
+    "start": "{start}"
+    "end":  "{end}"
     "part_type": "{part-type}",
 }
 ```
 
 Values:
-- **disk**: Full device path like `/dev/sda`.
-- **part-type**: Partition type as accepted by `parted mkpart`, e.g. `primary`
+- **disk**: Full device path like `sda`.
```
Comment (FastGeert) [inline docs/interacting/commands/disk.md]: why did you change it, it is not a full device path anymore

File: docs/interacting/commands/disk.md [hunk 2]
```diff
@@ -44,15 +44,17 @@ Arguments:
...
 - **start**: Partition start as accepted by `parted mkpart`, e.g. `1`
 - **end**: Partition end as accepted by `parted mkpart`, e.g. `100%`
+- **part-type**: e.g. `primary`,`extended` or `logical`
```
Comment (FastGeert) [inline docs/interacting/commands/disk.md]: Its ok to add more examples but why remove the specification?

**Comment by grimpy** [comment]
Lgtm

File: docs/interacting/commands/disk.md [hunk 3]
```diff
@@ -69,7 +71,7 @@ Arguments:
 ```
 
 Values:
-- **disk**: Full device path like `/dev/sda`.
+- **disk**: e.g. `sda`
```
Comment (FastGeert) [inline docs/interacting/commands/disk.md]: Is it not a devicepath anymore? Then you should at least say that it needs to be the device name. Then add an example

File: docs/interacting/commands/disk.md [hunk 4]
```diff
@@ -87,7 +89,7 @@ Arguments:
 ```
 
 Values:
-- **disk**: Full device path, e.g. `/dev/disk`
+- **disk**: e.g. `sda`
```
Comment (FastGeert) [inline docs/interacting/commands/disk.md]: Same here

File: docs/interacting/commands/zerotier.md [hunk 5]
```diff
@@ -2,10 +2,17 @@
 
 Available commands:
 
+- [zerotier.info](#info)
 - [zerotier.join](#join)
 - [zerotier.leave](#leave)
 - [zerotier.list](#list)
```
Comment (FastGeert) [inline docs/interacting/commands/zerotier.md]: Why did we remove these sections?

File: docs/interacting/python.md [hunk 6]
```diff
@@ -49,9 +49,29 @@ Some more simple examples:
...
+- Create a new partition on a given device:
+```python
+cl.disk.mkpart("sda", 1, "20%", "extended")
+```
+or
+```python
+cl.disk.mkpart("sda", 1, 400, "extended")
+```
+
+- List all disks or partitions:
```
Comment (FastGeert) [inline docs/interacting/python.md]: disks and partitions?

Comment (donatogomez) [inline docs/interacting/python.md]: Yes, my bad

File: docs/interacting/commands/zerotier.md (hunk 5 above)
Comment (donatogomez) [inline docs/interacting/commands/zerotier.md]: This is one of the possibles commands also for zerotier.

File: docs/interacting/commands/disk.md (hunk 2 above)
Comment (donatogomez) [inline docs/interacting/commands/disk.md]: I'll fix it

File: docs/interacting/commands/disk.md (hunk 3 above)
Comment (donatogomez) [inline docs/interacting/commands/disk.md]: Fix it!

File: docs/interacting/commands/disk.md (hunk 1 above)
Comment (donatogomez) [inline docs/interacting/commands/disk.md]: I changed it, because in the way that the documentation said, doesn't work. I forgot to change the specification. Is already fix it!

File: docs/interacting/commands/zerotier.md (hunk 5 above)
Comment (donatogomez) [inline docs/interacting/commands/zerotier.md]: image

Comment (FastGeert) [inline docs/interacting/commands/zerotier.md]: Yeah, but you removed join, leave and list

Comment (FastGeert) [inline docs/interacting/commands/zerotier.md]: My bad, I mis interpreted the diff.
